packages/sanity/src/core/config/types.ts

```diff
@@ -1003,7 +1003,7 @@ export interface BetaFeatures {
   }
 
   /**
-   * @beta
+   * @deprecated - start in create is no longer available.
```

Comment:
```suggestion
   * @deprecated - The Start in Create flow has been removed and will be updated in an upcoming studio release.
```

Just being a little more explicit here